CHANGELOG.md

```diff
@@ -23,12 +23,13 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 -   Fixed a bug where passing an empty array as objects will crash the program.
 -   Fixed a bug where the text may go outside of the box when it has a text font set
 
-### 📚 Documentation and demo website changes
+### 📚 Documentation, demo, and Webstepper website changes
```

Comment:
Since webstepper is really its own user-facing "product", please create a separate Changelog for it (`webstepper/CHANGELOG.md`) with the same format as this Changelog.